testutils/factories/referral.ts

```diff
@@ -26,7 +48,7 @@ class ReferralFactory extends Factory<Referral> {
   }
 
   assigned() {
-    return this.params({
+    return this.draft().params({
```

Comment:
Do we want this to be `this.sent()` rather than `draft()`? It doesn't make all that much difference in terms of data structure, but you can only assign a referral that has been "sent" so it might be clearer for people looking at the codebase.

Reply:
the issues with using the `sent()` constructor here is that you have to overwrite _all_ the sent fields because they are nested anyway.